pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1379,6 +1388,19 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     return path;
   }
 
+  protected async sudoRequiresPassword() {
+    try {
+      await childProcess.spawnFile('sudo', ['-n', 'true'], { stdio: ['ignore', 'pipe', 'pipe'] });
+      console.log('sudo -n didn\'t throw an error, so assume we can do passwordless sudo');
```

Comment:
```suggestion
      console.debug('sudo -n didn\'t throw an error, so assume we can do passwordless sudo');
```

And since you have `'`, use double quotes instead for the string delimiter?

Reply:
I thought our linter options don't allow that. I'll try to change it, and see what happens, but having to change the surrounding quotes back to single-quotes when you remove the last single quote from inside the string would be a really weird setting.